samples/api-client/accesstoken_example/README.rst.in

```diff
@@ -0,0 +1,25 @@
+# This file is used to generate README.rst
+
+product:
+  name: Google Cloud IoT Token Api
+  short_name: Cloud IoT Token Api
+  url: https://cloud.google.com/iot/docs
+  description: >
+    # Google Cloud IoT Core Python Device Federated Authentication example
+    This sample app demonstrates the capabilites of Google Cloud IoT Core device federated authentication feature.
+    Devices authenticated to Cloud IoT Core can use the [Token Service](https://cloud.google.com/iot/alpha/docs/reference/cloudiottoken/rest) federated authentication to request [OAuth 2.0 access tokens](https://developers.google.com/identity/protocols/oauth2) in exchange for their [Cloud IoT Core JWTs](https://cloud.google.com/iot/docs/how-tos/credentials/jwts).
```

Comment:
`[Token Service](https://cloud.google.com/iot/alpha/docs/reference/cloudiottoken/rest)`
This is not making a good use of the `rst` template style, see [this file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/927a5641395093a7a9b388ce7516d030d4c07e18/datastore/cloud-client/README.rst) on how to format the links better for `rst` templates.

Reply:
Why are we checking in the `.in` file here?  That file is part of the template, not the rendered output.

Reply:
@dandhlee Thanks for the heads up, links fixed.
@tseaver The `.in` files are checked in every repo, I think it is useful to have them checked in for later times when the samples changed.